packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -464,6 +464,7 @@ export class Profiles extends ProfilesCache {
                 if (choice === "project") {
                     rootPath = workspaceDir.uri.fsPath;
                     global = false;
+                    user = true;
```

Comment:
Would this cause a regression in behavior? In v2 we were accidentally creating a user config when someone selects the project option. This would bring back that behavior where it will create a user config instead of a normal config.